corehq/apps/integration/static/integration/js/hmac_callout.js

```diff
@@ -0,0 +1,86 @@
+/* globals CryptoJS */
```

Comment:
This should be included as a real dependency, should just be a matter of updating the `hqDefine` call to add `crypto-js/crypto-js` to the dependency list and `CryptoJS` to the callback's parameter list.

Reply:
I'm glad your back!

I tried to set things up that way, and had problems with it, I think it's because I wasn't sure if it was ok to add new "internal" dependencies to the `hqDefine` function specifically. 

If I wire things up that way, can I also remove the js static reference from the [FormPlayer Dependencies](corehq/apps/cloudcare/templates/formplayer/dependencies.html) file?

Reply:
Totally fine to add new dependencies to the `hqDefine` call. Were you running into errors, too? Let me know if you want me to take a look at it locally. The Crypto JS source looks like it should just work to add the dependency, but I've been wrong before.

You do have to leave the script tag, because web apps doesn't actually use requirejs yet. Adding the dependency just makes this module compatible with requirejs, saving work for whoever ends up migrating web apps. It also allows this module to be used on pages that **do** use requirejs, not that this module is likely to be used elsewhere.